docs/assets/documents/governance-for-hospitalNotification.md

```diff
@@ -0,0 +1,111 @@
+# Governance for MedCom HospitalNotification
+
+* [1 Exchange of HospitalNotification message](#1-exchange-of-hospitalnotification-message)
+  * [1.2 VANS-network and Fælleskommunal infrastructure](#12-vans-network-and-f-lleskommunal-infrastructure)
+    * [Type A](#type-a)
+    * [Type B](#type-b)
+    * [Type C](#type-c)
+  * [1.3 Acknowledgements](#13-acknowledgements)
+  * [1.4 Envelopes](#14-envelopes)
+* [2 HospitalNotification message](#2-hospitalnotification-message)
+  * [2.1 MedComHospitalNotificationMessage profile](#21-medcomhospitalnotificationmessage-profile)
+  * [2.2 MedComHospitalNotificationEncounter profile](#22-medcomhospitalnotificationencounter-profile)
+  * [2.3 MedComMessagingProvenance](#23-medcommessagingprovenance)
+<br><br>
+
+## 1 Exchange of HospitalNotification message 
+
+The flow of events of a HospitalNotification message differs from the regular message exchange over the VANS-network, as a message is exchange over the VANS-network and KOMBITs-network. Questions regarding KOMBITs infrastructure, called Fælleskommunal infrastructure, and KOMBIT's envelope, called Beskedfordeler-envelope, should be addressed to KOMBIT. 
+
+
+### 1.2 VANS-network and Fælleskommunal infrastructure
+The following sections describe the flow of a HospitalNotification message. There are three types of receiving systems that can receive a HospitalNotification: Type A, B and C. Since there is different requirements to acknowledging of a HospitalNotification, they are presented seperately.  
+
+The flow of events and rules, when sending and receiving a HospitalNotification, are common for them all systems. However, a type C system shall not acknowledge a HospitalNotification.
+
+|Rules describing the flow of events when sending, and acknowledging a HospitalNotification message|
+|:---|
+|A Sending EcoSystem **SHALL** wrap a HospitalNotification in a VANS-envelope|
+|The Fælleskommunal infrastructure **SHALL** unwrap a HospitalNotification in a VANS-envelope|
+|The Fælleskommunal infrastructure **SHALL** wrap a HospitalNotification in a Beskedfordeler-envelope|
+|A Receiving EcoSystem **SHALL** unwrap a HospitalNotification in a Kombit beskedfordeler kuvert|
+|A Receiving EcoSystem **SHALL** wrap an Acknowledgement in a Kombit beskedfordeler kuvert|
+|The Fælleskommunal infrastructure **SHALL** unwrap a Acknowledgement in a Beskedfordeler-envelope|
+|The Fælleskommunal infrastructure **SHALL** wrap a Acknowledgement in a VANS-envelope|
+|A Sending EcoSystem **SHALL** unwrap an Acknowledgement in a VANS-envelope|
+
+
+#### Type A
+A Type A system is according to KOMBIT a receiving system that also shall receive and acknowledge a HospitalNotification. The acknowledgement of a Type A system is the basis for the Fælleskommunal infrastructure to sent an Acknowledgement to the sending system. 
```

Comment:
A Type A system is according to KOMBIT a receiving system that also shall receive and acknowledge
erstattes af 
A Type A system is according to KOMBIT a receiving system that shall receive and acknowledge

Gælder også for Type B :-)